rome/src/test/resources/test.rome.properties

```diff
@@ -0,0 +1 @@
+test_property=part1 part2
```

Comment:
Maybe put test resources in the same package, i.e. `com.rometools.rome.io.impl`?